analyzers/tests/SonarAnalyzer.UnitTest/Helpers/ProjectConfigReaderTests.cs

```diff
@@ -103,49 +95,38 @@ public void WhenHasUnexpectedProjectType_FallsBackToProduct()
         [DataRow("MissingTargetFramework")]
         public void WhenHasMissingValues_FilesToAnalyzePath_ReturnsCorrectValue(string folder)
         {
-            var options = TestHelper.CreateOptions($"ResourceTests\\SonarProjectConfig\\{folder}\\SonarProjectConfig.xml");
-
-            var sut = new ProjectConfigReader(options);
+            var sut = CreateProjectConfigReader($"ResourceTests\\SonarProjectConfig\\{folder}\\SonarProjectConfig.xml");
 
             sut.FilesToAnalyzePath.Should().Be(@"c:\foo\bar\.sonarqube\conf\0\FilesToAnalyze.txt");
         }
 
-        [DataTestMethod]
-        [DataRow(null)]
-        [DataRow("/foo/bar/do-not-exit")]
-        [DataRow("/foo/bar/x.xml")]
-        public void WhenNoFileFound_ReturnsNull(string folder)
-        {
-            var options = TestHelper.CreateOptions(folder);
-
-            var sut = new ProjectConfigReader(options);
-
-            sut.AnalysisConfigPath.Should().BeNull();
```

Comment:
When there are multiple assertions I prefere using `AssertionScope`s  and that way be able to catch all the issues in a single run if there are more than one.

Reply:
That's +3 LOC for not too much of a value in this simple test (and it's siblings above). It will either fail  all because some common cause, or individually due to some renaming. It's easy to understand in both cases.